src/getDataFromMovieDB.js

```diff
@@ -10,10 +10,10 @@ export const getMoviePopular = async ({ page }) => {
 };
 export const getGenres = async () => {
     const response = await axios.get(`https://api.themoviedb.org/3/genre/movie/list?api_key=0a7a30a768304b21322288db13dcdb24&language=en-US`);
```

Comment:
We can build these URLs.
Many parts are repeated (e.g. baseURL or API key), so we can declare them above and then reuse them in the code.

Reply:
good idea :) I changed it in the code